enhancements/workload-partitioning/pao-render.md

```diff
@@ -0,0 +1,190 @@
+  judgement to determine the level of approval needed.  If you’re not sure,
+  just leave it blank and ask for input during review.
+
+# Performance Addon Operator render mode
+
+This is the title of the enhancement. Keep it simple and descriptive. A good
+title can help communicate what the enhancement is and should be considered as
+part of any review.
+
+The YAML `title` should be lowercased and spaces/punctuation should be
+replaced with `-`.
+
+To get started with this template:
+1. **Pick a domain.** Find the appropriate domain to discuss your enhancement.
+1. **Make a copy of this template.** Copy this template into the directory for
+   the domain.
+1. **Fill out the "overview" sections.** This includes the Summary and
+   Motivation sections. These should be easy and explain why the community
+   should desire this enhancement.
+1. **Create a PR.** Assign it to folks with expertise in that domain to help
+   sponsor the process.
+1. **Merge at each milestone.** Merge when the design is able to transition to a
+   new status (provisional, implementable, implemented, etc.). View anything
+   marked as `provisional` as an idea worth exploring in the future, but not
+   accepted as ready to execute. Anything marked as `implementable` should
+   clearly communicate how an enhancement is coded up and delivered. If an
+   enhancement describes a new deployment topology or platform, include a
+   logical description for the deployment, and how it handles the unique aspects
+   of the platform. Aim for single topic PRs to keep discussions focused. If you
+   disagree with what is already in a document, open a new PR with suggested
+   changes.
+1. **Keep all required headers.** If a section does not apply to an
+   enhancement, explain why but do not remove the section. This part
+   of the process is enforced by the linter CI job.
+
+The `Metadata` section above is intended to support the creation of tooling
+around the enhancement process.
+
+## Release Signoff Checklist
+
+- [x] Enhancement is `implementable`
+- [ ] Design details are appropriately documented from clear requirements
+- [ ] Test plan is defined
+- [ ] Operational readiness criteria is defined
+- [ ] Graduation criteria for dev preview, tech preview, GA
+- [ ] User-facing documentation is created in [openshift-docs](https://github.com/openshift/openshift-docs/)
+
+## Summary
+
+The Performance addon operator is a day 2 optional operator installed via OLM. However the [Workload partitioning](management-workload-partitioning.md) feature needs to have some configuration ready at the installation time when PAO is not yet ready.
+This feature is about adding a `render` mode to PAO that would allow an admin to pre-compute all the Openshift manifests that are needed in a way that prevents any typing mistakes from happening.
+These generated manifests can then be passed to the installer and later taken over by the PAO reconcile loop. 
+
+## Motivation
```

Comment:
In addition to providing a mechanism to enable to workload partioning feature, the application of the manifests at install time will reduce the e2e deployment time for solutions that use pao as there will be fewer reboots. Suggest we capture that as a motivation as well.

Reply:
Based on some of the review feedback on #753, are we sure this will be true immediately? We may need to do some additional work in the MCO to get that benefit. We should still capture it as a goal, but not lose sight of the gap.